articles/i2p-docker-services/GithubPagesMirror.md

```diff
@@ -0,0 +1,233 @@
+Mirroring an existing Github Pages site using a Static eepSite
+==============================================================
+
+This is kind of a softball tutorial, it's almost the same as the **[Previous tutorial](BasicStaticeepSite.md):**
+but it's intended specifically to allow you to mirror a Github Pages based site
+using Jekyll to i2p using i2pd and Tor. Of course, this means that the
+eepSite is by definition linked to your identity on Github, but this might be
+desirable, for instance for i2p applications that collaborate primarily by way
+of Github, but who wish to have a presence on i2p as well. Probably other cases
+where long-term pseudonymity is desirable.
+
+So, just to be clear
+--------------------
+
+  * *For this tutorial to be unlinkable to your real identity, your github*
```

Comment:
If you remove the newline seperating the two parts of the dot point it will render better on the site (linebreaks may not match up with screen size, so it can look weird).

Reply:
eg:

* *Sunt enim qui vel consequatur provident qui expedita. Velit ratione repudiandae distinctio consequuntur. Quasi corrupti* 
*qui illo ducimus aliquid sint non. Fugiat illum dolores est dolor. Sunt et molestiae nostrum. Et sit possimus eum.*

VS

* *Sunt enim qui vel consequatur provident qui expedita. Velit ratione repudiandae distinctio consequuntur. Quasi corrupti qui illo ducimus aliquid sint non. Fugiat illum dolores est dolor. Sunt et molestiae nostrum. Et sit possimus eum.*